docs/design/plugin_structure.md

```diff
@@ -27,20 +26,21 @@ Initialize processing is like the following.
 4. Initialize the plugin according to the merged config, which is created by the input config and the default config.
 
 
+
 ## Plugin usage in Satellite
+Nowadays, we have 9 kinds of plugins in Satellite. 2 kinds of Plugins are the [sharing plugins](module_design.md) and the others are the [normal plugins](module_design.md).
```

Comment:
The word `kinds` is over-used. Consider using `Catalog S` for sharding plugins and `Catalog N` for normal plugins?